core/jobs/batch_jobs/firebase_sync_jobs.py

```
@@ -0,0 +1,142 @@
+"""Job for bringing Firebase records in sync with Oppia's user & auth models."""
+
+from __future__ import annotations
+
+from core.domain import feature_flag_domain
+from core.jobs import base_jobs
+from core.jobs.io import firebase_io
+from core.jobs.transforms import firebase_transforms, job_result_transforms
+from core.jobs.types import firebase_adapters, job_run_result
+
+import apache_beam as beam
+from typing import Iterable, TypedDict
```

Ruff (0.15.15)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)